tokenizers/src/normalizers/strip.rs

```diff
@@ -36,3 +37,91 @@ impl Normalizer for Strip {
         Ok(())
     }
 }
+
+/// This normalizer removes combining marks from a normalized string
+/// It's different from unidecode as it does not attempt to modify
```

Comment:
nit: I don't know what is `unidecode` in this context, and it's probably irrelevant to have this in the doc.

Reply:
Unidecode is a way to get ~ascii from a unicode string:
https://pypi.org/project/Unidecode/
https://github.com/chowdhurya/rust-unidecode

I put it here, because googling "Strip accents unicode" yield unidecode as a first answer, and could be a source of confusion for users/readers. Could be a regular comment instead of doc comment.